p2p/discovery/distance_test.go

```diff
@@ -0,0 +1,52 @@
+/**
+*  @file
+*  @copyright defined in go-seele/LICENSE
+ */
+package discovery
+
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+	"github.com/seeleteam/go-seele/common"
+	"github.com/seeleteam/go-seele/common/hexutil"
+	"github.com/seeleteam/go-seele/log"
+)
+
+func Test_Dic(t *testing.T) {
+	h1 := getHash("0x426225605450f355f558f89df996c0025223f08b056354d3471d64313c506dfd")
+	h2 := getHash("0x6bb28d28c1a01c62ffec43dcdee6dbd2b3d4822b0301593068b62c63b49fe358")
+	h3 := getHash("0x3d7c5dad1d99e0fce26ecefdd8904f304d979ff424ad666832544b72851e4b52")
+	h4 := getHash("0x426225605450f355f558f89df996c0025223f08b056354d348b62c63b49fe358")
+
+	log1 := logdist(h1, h2)
+	log2 := logdist(h1, h3)
+	log3 := logdist(h1, h1)
+	log4 := logdist(h1, h4)
+
+	assert.Equal(t, log3, 0)
```

Comment:
suggest to add assert for log1/2/3/4.

better and comprehensive test cases should be covered all pairs:
1-1, 1-2, 1-3, 1-4
2-1, 2-2, 2-3, 2-4
.......

Reply:
yes, I will refine this test later